src/journal/journal-send.c

```diff
@@ -73,33 +73,52 @@ _public_ int sd_journal_print(int priority, const char *format, ...) {
 }
 
 _public_ int sd_journal_printv(int priority, const char *format, va_list ap) {
-
-        /* FIXME: Instead of limiting things to LINE_MAX we could do a
-           C99 variable-length array on the stack here in a loop. */
-
-        char buffer[8 + LINE_MAX], p[STRLEN("PRIORITY=") + DECIMAL_STR_MAX(int) + 1];
-        struct iovec iov[2];
+        char p[STRLEN("PRIORITY=") + DECIMAL_STR_MAX(int) + 1];
+        int len, i, nlines;
+        _cleanup_free_ char *tbuffer;
```

Comment:
This needs to be initialized with `= NULL`.

Reply:
Removed dynamic memory allocation.